radxa.c

```diff
@@ -312,9 +314,37 @@ static int changeOwner(char *file) {
 	return 0;
 }
 
+static int radxaKernelRev(void) {
+	FILE *versionFd;
+	char line[120];
+
+	if((versionFd = fopen("/proc/version", "r")) == NULL) {
+		wiringXLog(LOG_ERR, "radxa->identify: Unable open /proc/version");
+		return -1;
+	}
+
+	fgets(line, 120, versionFd);
+	fclose(versionFd);
+
+	if(strstr(line, "3.0.36") != NULL) {
```

Comment:
I would suggest checking the actual versions as numbers through a `sscanf` and then something like:
`if(major_ver <= 3 && minor_ver < 18 && patch_ver <= 0)`.